### ISSUE TITLE
[SF][Table]: when focus on a table row, the row's content was read out by screen reader but not associated with column headers

### ISSUE THREAD

**Issue by summer2200** [SAP/ui5-webcomponents-react#6775]
### Describe the bug

With screen reader turned on,

Issue:
When table data rows are focused, only row data get read out.

Expected:
When table data rows are focused, corresponding column header should get associated and read out along with row data.

### Reproduction steps

1.
2.
3.
...

### UI5 Web Components for React Version

2.5.0

### UI5 Web Components Version

2.5.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
